packages/core/src/routes/sign-in-settings.ts

```diff
@@ -1,19 +1,28 @@
 import { ConnectorMetadata } from '@logto/connector-types';
+import { SignInMode } from '@logto/schemas';
+import { adminConsoleApplicationId, adminConsoleSignInMethods } from '@logto/schemas/lib/seeds';
 import etag from 'etag';
+import { Provider } from 'oidc-provider';
 
 import { getConnectorInstances } from '@/connectors';
 import { findDefaultSignInExperience } from '@/queries/sign-in-experience';
+import { hasActiveUsers } from '@/queries/user';
 
 import { AnonymousRouter } from './types';
 
-export default function signInSettingsRoutes<T extends AnonymousRouter>(router: T) {
+export default function signInSettingsRoutes<T extends AnonymousRouter>(
+  router: T,
+  provider: Provider
+) {
   router.get(
     '/sign-in-settings',
     async (ctx, next) => {
-      const [signInExperience, connectorInstances] = await Promise.all([
+      const [signInExperience, connectorInstances, interactions] = await Promise.all([
```

Comment:
should be singular: interactions → interaction

Reply:
yep updated